src/pkg/user/manager.go

```diff
@@ -116,7 +116,11 @@ func (m *manager) MatchLocalPassword(ctx context.Context, usernameOrEmail, passw
 	return nil, nil
 }
 
-func (m *manager) Count(ctx context.Context, query *q.Query) (int64, error) {
+func (m *manager) Count(ctx context.Context, query *q.Query, options ...models.Option) (int64, error) {
+	opts := models.NewOptions(options...)
+	if !opts.IncludeDefaultAdmin && query != nil && query.Keywords != nil {
```

Comment:
If the opts.IncludeDefaultAdmin is false and the query is nil, seems this doesn't work as expected.
How about changing it as following?
```go
if !opts.IncludeDefaultAdmin {
  if query == nil {
    // new a new query
 }
if query.Keywords == nil {
  // new keywords
}
}
```

Reply:
ok, updated